src/index.ts

```diff
@@ -12,10 +12,13 @@ appInsights.setup(clientKey)
     .setUseDiskRetryCaching(true)
     .setSendLiveMetrics(false)
     .setDistributedTracingMode(appInsights.DistributedTracingModes.AI_AND_W3C)
-    .start();
+    
 
-export const aiClient = appInsights.defaultClient
 export const ai = appInsights // in case you need to override setup()
+export const aiClient = appInsights.defaultClient
+aiClient.config.samplingPercentage = parseInt(process.env.SAMPLING_PERCENTAGE || '100')
```

Comment:
Lets rename this env var to `AI_SAMPLING_PERCENTAGE` to keep consistent with our other config vars

Reply:
No problem. Also, there is more to this - sorry for the early tag for review.